spine-contraster.html

```diff
@@ -0,0 +1,152 @@
+<link rel="import" href="../polymer/polymer-element.html">
+
+<script>{
+    /**
+     * `SpineContraster` is a mixin to determine to which dark or light theme an element belongs
```

Comment:
Let's use the same as for `README.md`, `determines the theme (dark or light) an element belongs to based on its background color`, what do you think?